clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -592,9 +593,32 @@ val3
 
 #[test]
 fn suggest_custom_arg_value() {
-    let mut cmd = Command::new("dynamic").arg(clap::Arg::new("custom").long("custom"));
+    #[derive(Debug)]
+    struct MyCustomCompleter {}
+
+    impl CustomCompleter for MyCustomCompleter {
+        fn custom_hint(&self) -> Vec<CompletionCandidate> {
+            vec![
+                CompletionCandidate::new("custom1").visible(true),
```

Comment:
Hmm, is `visible(false)` the default?  I wonder if we should swap that to instead have `hidden(false)`  or `hide(false)` (whichever maps to clap's API better).

My thinking is that visible completions will be the more common case and it will be easy for people to leave this off and be confused with the result.

Reply:
Thanks for your reminder! I will reflect this small change in a separate commit.

Reply:
We should also rename `visible` to [`hide`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.hide)